xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml

```diff
@@ -661,18 +659,27 @@ span#avatarUpload {
   margin: 0;
 }
 
-.attachment-picker-start {
-  background: url("$xwiki.getSkinFile('icons/silk/picture_edit.png')") no-repeat center center $theme.pageContentBackgroundColor !important;
-  border: 0 none !important;
-  border-bottom-left-radius: 8px;
-  height: 18px;
-  position: absolute;
-  right: 0;
-  text-align: left;
-  text-indent: -9999px;
-  top: 1px;
-  width: 18px !important;
-  z-index: 1;
+.attachment-picker {
+  .buttonwrapper {
+    a.button {
+      background-color: inherit;
+      background-image: none;
+      border-color: unset;
+      border-radius: unset;
+      border: unset;
+      color: @link-color;
+      cursor: pointer;
+      display: unset;
+      font-size: @font-size-small;
+      font-weight: normal;
+      line-height: unset;
+      margin: unset;
+      padding: unset;
+      text-align: center;
+      vertical-align: middle;
+      &amp;:hover{text-decoration: underline};
```

Comment:
You should add some spaces and newlines to match the codestyle :)
See for example https://github.com/xwiki/xwiki-platform/blob/92c46a63a954990fb64e95b78a4a4e112cb1583a/xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml#L590-L592

Reply:
Solved by [6a9947e](https://github.com/xwiki/xwiki-platform/pull/2862/commits/6a9947e5503b0b67f192e300093a279f9ae50181) :+1: